download/index.md

```diff
@@ -3,72 +3,48 @@ title: Download
 
 ```bash
-docker run -ti -p 127.0.0.1:8000:8000 -v $(pwd):/home/fenics/shared -w /home/fenics/shared quay.io/fenicsproject/stable:current
+docker pull dolfinx/lab
 ```
 
-For detailed instructions, see the [FEniCS Reference Manual](http://fenics-containers.readthedocs.io/en/latest/index.html).
-
-## FEniCS on Windows 10
-To install FEniCS on Windows 10, enable the [Windows Subsystem for Linux](https://docs.microsoft.com/en-us/windows/wsl/install-win10)
-and install the Ubuntu distribution. Then follow the instructions for Ubuntu below.
-
-## Ubuntu FEniCS on Ubuntu
-To install FEniCS on Ubuntu, run the following commands:
+The image can then be started by running:
 
 ```bash
-sudo apt-get install software-properties-common
-sudo add-apt-repository ppa:fenics-packages/fenics
-sudo apt-get update
-sudo apt-get install fenics
+docker run -p 8888:8888 dolfinx/lab
 ```
 
-For detailed instructions, see the [FEniCS Reference Manual](http://fenics-containers.readthedocs.io/en/latest/index.html).
-
-## FEniCS on Anaconda
-To use our prebuilt Anaconda Python packages (Linux and Mac only), first
-[install Anaconda](https://docs.continuum.io/anaconda/install), then run following commands in
-your terminal:
+A URL will then be output in the terminal that leads to the Jupyter lab. From there, Jupyter notebooks and
+Python scripts can be created and run.
 
+## Developing FEniCSx using the Docker dev-env
+If you want to run FEniCSx from a Docker image while developing the source code, you can use the FEniCSx
+development environment image. This image contains all the dependencies of the FEniCSx libraries. It can
+be downloaded by running:
 
 ```bash
-conda create -n fenicsproject -c conda-forge fenics
-source activate fenicsproject
+docker pull dolfinx/dev-env
 ```
 
-For further information on using Anaconda, see the [documentation](https://docs.continuum.io/anaconda/).
-
-Installing FEniCS via Anaconda is also supported in [Microsoft Azure Notebooks](https://notebooks.azure.com/).
-In the first cell of a new Jupyter notebook, type:
+You can then build your own version of the FEniCSx libraries inside this image. For example, if your copies
+of the source code of UFL, Basix, FFCx and DOLFINx are stored in `/home/username/development/`, you can
+run the Docker image with these folders shared by running:
 
-```python
-!conda config --add channels conda-forge
-!conda install fenics
+```bash
+docker run -ti -v /home/username/development/:/mnt/development dolfinx/dev-env
 ```
 
-**Warning**: FEniCS Anaconda recipes are maintained by the community and distributed binary 
-packages do not have a full feature set yet, especially regarding sparse direct solvers and 
-input/output facilities.
-
-**Update**. 2017.2.0 release on `conda-forge` features MUMPS direct solver, but lacks SuperLU_dist and MPI-enabled HDF5.
+Inside the Docker image, your source code folder will be available at `/mnt/development`. You can navigate
+to that folder and build the components.
```

Comment:
Shall we add Spack packages (which would be a duplicate of what's in the dolfinx readme)?

Reply:
I think we should cut almost all of install instructions and point to the docs (and improve the docs). Otherwise we have duplication and it will become inconsistent.

Reply:
See also https://github.com/FEniCS/web/issues/71 (although it's suggesting the wrong thing if we want the instructions in the docs instead)

Reply:
I've edited it to just have instructions for how to run the lab docker image and then a link to https://docs.fenicsproject.org/dolfinx/main/python/installation.html for further instructions